test/gateway/eip20_gateway/revert_stake.js

```diff
@@ -52,9 +66,254 @@ contract('EIP20Gateway.revertStake()', function (accounts) {
       burner,
     );
 
+    let hashLockObj = Utils.generateHashLock();
+
+    stakeMessage.hashLock = hashLockObj.l;
+    stakeMessage.unlockSecret = hashLockObj.s;
+    stakeMessage.messageHash = messageBus.messageDigest(
+      stakeMessage.intentHash,
+      stakeMessage.stakerNonce,
+      stakeMessage.gasPrice,
+      stakeMessage.gasLimit,
+      stakeMessage.staker,
+      stakeMessage.hashLock,
+    );
+
+    await gateway.setStake(
+      stakeMessage.messageHash,
+      stakeRequest.beneficiary,
+      stakeRequest.stakeAmount,
+    );
+    await gateway.setStakeMessage(
+      stakeMessage.messageHash,
+      stakeMessage.intentHash,
+      stakeMessage.stakerNonce,
+      stakeMessage.gasPrice,
+      stakeMessage.gasLimit,
+      stakeMessage.staker,
+      stakeMessage.hashLock,
+    );
+
+  });
+
+  it('should emit event on success', async function () {
+
+    let penalty = new BN(150);
+
+    await baseToken.approve(gateway.address, penalty, {from: stakeMessage.staker});
+    await gateway.setOutboxStatus(
+      stakeMessage.messageHash,
+      MessageStatusEnum.Declared,
+    );
+
+    let tx = await gateway.revertStake(
```

Comment:
Add asserts for `gateway.revertStake.call` returned values.
or create a new `it` for this .